### ISSUE TITLE
Add Support for actual piracy websites

### ISSUE THREAD

**Issue by FormunaGit** [Wamy-Dev/ReziWebsite#22]
Y'know what I'm talking about. TheTorrentBay, RARBG, etc...

**Comment by antunnitraj** [comment]
Those sites are full of "non game" content and actual viruses.

**Comment by Wamy-Dev** [comment]
Likely won't add torrent trackers because Rezi is mostly focused on DDL rather than torrents. If more people think it's a good idea I can add it.

**Comment by antunnitraj** [comment]
What about freegogpcgames.com? They are hosting torrents for gog games (like gog-games.to) so its faster to download. Its recommended by r/FREEMEDIAHECKYEAH on their website (GOG Games - Torrent).

**Comment by Wamy-Dev** [comment]
> What about freegogpcgames.com? They are hosting torrents for gog games (like gog-games.to) so its faster to download. Its recommended by r/FREEMEDIAHECKYEAH on their website (GOG Games - Torrent).

that site just looks sketchy, definitely not that one

**Comment by FormunaGit** [comment]
> Rezi is mostly focused on DDL rather than torrents.

There are lots of DDL piracy websites, so you could use those.
You can also add badges next to the names so you can see if it's from 3 categories: Retro (MyAbandonware, Archive.org), DDL (steamunlocked), and Torrent.
On the topic of categorizing, are you thinking on adding filters?

EDIT: Forgot to tell you about the champagne wiki, it's a wiki filled with piracy guides.
https://champagne.pages.dev/

**Comment by Wamy-Dev** [comment]
> > Rezi is mostly focused on DDL rather than torrents.
> 
> There are lots of DDL piracy websites, so you could use those. You can also add badges next to the names so you can see if it's from 3 categories: Retro (MyAbandonware, Archive.org), DDL (steamunlocked), and Torrent. On the topic of categorizing, are you thinking on adding filters?
> 
> EDIT: Forgot to tell you about the champagne wiki, it's a wiki filled with piracy guides. [champagne.pages.dev](https://champagne.pages.dev/)

Rezi already has most of the popular sites, if you want, you can open a pull request with scrapers. I don't really work on Rezi anymore, although I do provide updates and support, not much else past that. Writing a scraper is pretty easy, just copy an existing one, change the url, and change the css selectors.

Rezi already has basic filters thanks to Meilisearch. If you want to search for a specific console, just search for that console before the game, or just search the console. If you want to search for *fitgirl* repacks just search for that.

TLDR: I likely won't be developing Rezi anymore so if you want to add sites, you are free to open a pr, I'll review and add it.

**Comment by FormunaGit** [comment]
Note taken, might start making pull requests, I think I'll start with updating the icons.
